### PR TITLE
Changes intake from using two buttons to one

### PR DESCRIPTION
**Pull request by Involuta**
IntakeCommand will now run constantly, but only be functional when the Y button is pressed. This button can and will be changed later depending on the driver's preferred controls; the Y button feature is simply a placeholder.

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/IntakeSubsystem.java [hunk 1]
```diff
@@ -38,7 +38,7 @@ public void intakeSpin(double percentVoltage) {
 
   @Override
   public void initDefaultCommand() {
-    // Set the default command for a subsystem here.
+    setDefaultCommand(new IntakeCommand());
```
Comment (kozzza) [inline src/main/java/frc/robot/subsystems/IntakeSubsystem.java]: don't set as default command

File: src/main/java/frc/robot/OI.java [hunk 2]
```diff
@@ -37,14 +36,11 @@ public OI() {
...
-    rightBumperButton.whileHeld(new IntakeCommand());
-
-    Button leftBumperButton = new JoystickButton (stick, RobotMap.leftBumperPort);
-    leftBumperButton.whileHeld(new ReverseIntakeCommand());
+    //Button rightBumperButton = new JoystickButton(stick, RobotMap.rightBumperPort);
+    //rightBumperButton.whileHeld(new IntakeCommand());
 
     Button buttonXButton = new JoystickButton(stick, RobotMap.buttonXPort);
-    buttonXButton.whenPressed(new PneumaticCommand());
+    buttonXButton.whenPressed(new PneumaticShooterCommand());
```
Comment (kozzza) [inline src/main/java/frc/robot/OI.java:43]: instead of whenPressed use whilelHeld

File: src/main/java/frc/robot/commands/IntakeCommand.java [hunk 3]
```diff
@@ -28,7 +31,14 @@ protected void initialize() {
...
+    //you can replace the Y Button input with whatever the driver feels like. No command calls were added in the OI.
+    if (Robot.oi.stick.getYButton() == false) {
+      intakeInput= 0;
+    }
+    else if (Robot.oi.stick.getYButton() == true) {
+      intakeInput = 1;
+    }
+    Robot.intakeSubsystem.intakeSpin(0.5f * intakeDirection * intakeInput);
     
   }
```
Comment (kozzza) [inline src/main/java/frc/robot/commands/IntakeCommand.java:37]: fix logic here to comply with whileHeld

**Review by kozzza — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/IntakeSubsystem.java (hunk 1 above)
Comment (Involuta) [reply src/main/java/frc/robot/subsystems/IntakeSubsystem.java]: why can't the intake command be set as a default command? the intake will not receive voltage unless a certain button is held, negating the need for using whileHeld in the OI

Comment (kozzza) [reply src/main/java/frc/robot/subsystems/IntakeSubsystem.java]: this is a redundancy and you don't want to set to default command unless absolutely necessary. This means the command will be constantly run in every loop. If you can reduce the amount of stuff being iterated through, you effectively increase efficiency. Also whileHeld is just a better way to code it for this purpose.

**Review by kozzza — APPROVED** [review]

**Pull request merged**